src/gameProtocol.js

```diff
@@ -11,7 +11,7 @@ export const gameStateEventArgs = {
     { id: '', username: '', piece: O },
   ],
   playerTurn: '',
-  state: gameStatus.STARTED,
+  state: GAME_STATUS.STARTED,
   board: [EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY],
 }
```

Comment:
As stated [here](https://github.com/tic-tac-toe-devs/tic-tac-toe-backend/pull/24#discussion_r988628147):
I see why we might want to have the arguments laid out as such. But since they are placeholders and used for illustrative purposes only, they should be named as such. For example: mockGameStateEventArgs